src/Config.php

```diff
@@ -16,6 +16,8 @@ public function __construct()
 
     public function getRules(): array
     {
-        return [];
+        return [
+            '@PSR2' => true,
+        ];
```

Comment:
There are a couple of PSR2 & PSR1 rules that we ignore in most repo's:

```
<rule ref="PSR2">
    <exclude name="Generic.Files.LineLength"/>
</rule>

<rule ref="PSR1.Methods.CamelCapsMethodName.NotCamelCaps">
    <exclude-pattern>*/tests/*</exclude-pattern>
</rule>
<rule ref="PSR1.Classes.ClassDeclaration.MultipleClasses">
    <exclude-pattern>*/tests/*</exclude-pattern>
</rule>
```

- https://github.com/cultuurnet/udb3-silex/blob/master/phpcs.xml.dist#L14-L23
- https://github.com/cultuurnet/udb3-search-service/blob/master/phpcs.xml.dist#L14-L23
- https://github.com/cultuurnet/jwt-provider/blob/master/phpcs.xml.dist#L14-L23

The `Generic.Files.LineLength` was excluded because it results in a lot of warnings that obfuscate actual errors and other warnings.

`PSR1.Methods.CamelCapsMethodName.NotCamelCaps` was excluded because the test method names are now `it_should_...()`. I'd be okay with changing that to `itShould...()` though, but just pointing it out might require a lot of changes. If it could be done automatically it would be easy to do of course.

`PSR1.Classes.ClassDeclaration.MultipleClasses` is probably because of some test files with multiple classes, maybe because in the past it wasn't clear we could just autoload the classes in the tests directory as well? Would also be good to clean that up if there are any left, but I can't say how many classes it would be.

I mostly just wanted to point this out and let you decide if we should merge this and then see if we need to make changes, or if you want to exclude some already like the `LineLength`.

Reply:
Unrelated: We also enforce `Symfony.Arrays.MultiLineArrayComma` in most repo's already, maybe we should include that here from the start so we don't forget?